models/plan_statistics.py

```diff
@@ -0,0 +1,56 @@
+from models.refugee import Refugee
+from models.volunteer import Volunteer
+from models.camp import Camp
+from models.plan import Plan
+from datetime import date
+import math
+
+
+def find_volunteers(Camp):
+    """
+    Function to find the number of active volunteers at a camp. If a volunteer is not active then they will not be included in the count.
+    """
+    volunteer_count = 0
+    for volunteer in Camp.volunteers:
+        if volunteer.availability == True and volunteer.account_activated == True:
+            volunteer_count += 1
+    return volunteer_count
+
+
+def find_refugees(Camp):
+    """
+    Function to find the number of refugees at a camp. Includes the count of both the head of family and the family members in a single count.
+    """
+    refugee_count = 0
+    for refugee in Camp.refugees:
+        refugee_count += refugee.num_of_family_member
+    return refugee_count
+
+
+def plan_statistics_function(Plan):
```

Comment:
Thank you! I will strongly consider this if I have time to change it before our Friday deadline

Reply:
Maybe I was not clear, but what I mean is that since this is a programming project, I believe we should write this function in the classes, since the organisation of our code will very likely count towards our mark. 
Moreover, since I have explained all the changes in detail, and also wrote sample codes, I think that it should be manageable. 
Please raise to the group if you need assistance on refactoring the code.

Reply:
Thank you! Elynor and I have created a new plan statistics branch and have made a pull request. The updates in the new branch are moving plan statistics to camp and plan classes, modifying functions to pure functions, and the comments for cleaning up the code. Please advice for any possible improvements!